extensions/interactions/TextInput/directives/text-input-validation.service.ts

```diff
@@ -76,15 +76,17 @@ export class TextInputValidationService {
       let textSpecs = InteractionSpecsConstants.INTERACTION_SPECS.TextInput;
       let customizationArgSpecs = textSpecs.customization_arg_specs;
       let rowsSpecs = customizationArgSpecs[1];
-      let minRows = rowsSpecs.schema.validators[0].min_value;
-      let maxRows = rowsSpecs.schema.validators[1].max_value;
-      if (rows < minRows || rows > maxRows) {
-        warningsList.push({
-          type: AppConstants.WARNING_TYPES.ERROR,
-          message: (
-            'Number of rows must be between ' + minRows + ' and ' +
-            maxRows + '.')
-        });
+      if (rowsSpecs.schema.validators) {
```

Comment:
defining a variable for this and using it all the places will decrease redundancy

Reply:
done